dotcom-rendering/src/components/FrontMostViewed.tsx

```diff
@@ -91,7 +91,7 @@ export const FrontMostViewed = ({
 			) : (
 				<MostViewedFooter
 					tabs={tabs}
-					sectionName="Most viewed"
+					sectionId="Most viewed"
```

Comment:
This one is a bit odd. For articles we really do pass the sectionId into `MostViewedFooter`, but for fronts we pass in "Most viewed".
It's actually only used to build a `data-link-context` attribute in a child component.